src/it/scala-2.12/com/linagora/gatling/imap/JamesServer.scala

```diff
@@ -37,7 +38,13 @@ object JamesServer extends Server {
   }
 
   def start(): RunningServer = {
-    val james = new GenericContainer("linagora/james-memory")
+    val james = new GenericContainer("linagora/tmail-backend:memory-0.4.3")
```

Comment:
Why not branch-master?

Reply:
It could be unstable at some time?

Reply:
I'm afraid we forget to upgrade it again for a long time after this^^'

Reply:
IMO we should update the version manually. The webadmin host issue was an example. By doing it manually we could know where is the problem came from by looking at the version diff.

Reply:
Could this be acceptable to you, @Arsnael ?

Reply:
That's what I mean by manually... Easy to just forget to update it :P And then you realise at some point you are running on an x years old image :D 

I would think the other way around: the image being updated regularly if at some points it breaks it should be easy enough to identify the issue and fix it based on recent codebase... VS if we forget it might be trickier to remember

@chibenwa , any thoughts?

Reply:
IMO you both have a point

 - Using fixed docker tags is a best practice...
 - But having to update it is inconvenient and we tend to forget.

>  the image being updated regularly if at some points it breaks it should be easy enough to identify the issue and fix it based on recent codebase... 

True if we have a build with some alerting emails sent. I'm not fully sure it is the case.

I remember we had some travis build, but maybe it just is for https://github.com/linagora/james-gatling ....

If we can setup a build (Jenkins or travis) and ensure we know when/if the build fails then yes the `branch-master` stuff would definitly be worth it.

IMO adding a Jenkins build here would be a good way to spread knowledge about Jenkins, and solve this discussion as well.